waspc/data/Generator/templates/server/src/auth/providers/email/resetPassword.ts

```diff
@@ -1,24 +1,35 @@
 import { Request, Response } from 'express';
-import { findUserBy, verifyToken } from "../../utils.js";
-import { updateUserPassword } from "./utils.js";
+import {
+    findAuthIdentityByAuthId,
+    updateAuthIdentityProviderData,
+    verifyToken,
+    deserializeProviderData,
+} from "../../utils.js";
 import { ensureTokenIsPresent, ensurePasswordIsPresent, ensureValidPassword } from "../../validation.js";
 import { tokenVerificationErrors } from "./types.js";
 
 export async function resetPassword(
     req: Request<{ token: string; password: string; }>,
     res: Response,
 ): Promise<Response<{ success: true } | { success: false; message: string }>> {
-    const args = req.body || {};
+    const args = req.body ?? {};
     ensureValidArgs(args);
 
     const { token, password } = args;
     try {
-        const { id: userId } = await verifyToken(token);
-        const user = await findUserBy({ id: userId });
-        if (!user) {
+        const { id: authId } = await verifyToken(token);
```

Comment:
> I thought here that authId is id of instance of Auth model, but it seems from code below like it is not

`authId` === `Auth.id` that's correct 👍

Reply:
Ok got it - what confused me was `findAuthIdentityByAuthId` function, because I couldn't figure out how we can find a specific auth identity by just an auth id, since here should be multiple ones connected to one auth id -> but the problem here is a bad name of that function, which I commented on separately.

Reply:
This function is now gone 👍